src/test/resources/expectedOutcomes/8.3.0-alpha13_outcomes.json

```diff
@@ -35,10 +35,10 @@
     ":app:extractReleaseNativeSymbolTables"              :  "NO_SOURCE",
     ":app:extractReleaseVersionControlInfo"              :  "SUCCESS",
     ":app:generateDebugAssets"                           :  "UP_TO_DATE",
-    ":app:generateDebugResources"                        :  "UP_TO_DATE",
+    ":app:generateDebugResources"                        :  "FROM_CACHE",
```

Comment:
@cdsap do you think this is worth reporting to Google?

Reply:
yep, sounds good
let me check what's there in the task

Reply:
some findings, there is a change in the type of task used for `generate$BuildTypeResources`:
* In `8.3.0-alpha12` the type was DefaultTask, I assume registered during the plugin configuration: https://ge.solutions-team.gradle.com/s/quh4ztwynlohc/timeline?name=generateDebugResources 
* In `8.3.0-alpha14` the type is `ValidateResourcesTasks` https://ge.solutions-team.gradle.com/s/dy2sthbz4oqd2/timeline?name=generateDebugResources  More info about the [Task](https://cs.android.com/android-studio/platform/tools/base/+/mirror-goog-studio-main:build-system/gradle-core/src/main/java/com/android/build/gradle/internal/tasks/ValidateResourcesTask.kt;l=44?q=validatereso)

Reply:
Thanks for investigate. In that case, it seems this change is intentional.